packages/@expo/config-plugins/src/android/Package.ts

```diff
@@ -206,7 +206,11 @@ export async function renamePackageOnDiskForType({
     try {
       if (fs.lstatSync(filepath).isFile()) {
         let contents = fs.readFileSync(filepath).toString();
-        contents = replacePackageName(contents, currentPackageName, packageName);
+        if (path.extname(filepath) === '.kt') {
+          contents = replacePackageName(contents, currentPackageName, kotlinSanitized(packageName));
```

Comment:
Let's lift `kotlinSanitized` out of the loop. We only need to do this once. Also, it would be great if we could add some test coverage on this function?

Reply:
I can pull this out. Could I get some guidance on how to add some test for this?

Reply:
Sure, you can add tests to this file https://github.com/expo/expo/blob/main/packages/%40expo/config-plugins/src/android/__tests__/Package-test.ts Just some coverage passing various package name variants to the `kotlinSanitized` would be great.

Reply:
We could also remove these lines https://github.com/expo/expo/blob/73d530d95d8a7637c2f1f93b2b1bc6449139c02a/packages/%40expo/config-plugins/src/android/Package.ts#L259-L260 as I don't think they are working correctly but if you run into problems with tests after this you can leave this

Reply:
Pushed some commits, I think that should be 👌, but I wanna wait for tests to run on ci.

Reply:
Op, apparently removing these two lines breaks the test `package › renamePackageOnDisk › re-creates the directory structure and replaces occurrences of old package in files`

Reply:
That's fine, we can leave that then and we'll take a look at it 👍